dicompyler/baseplugins/2dview.py

```diff
@@ -47,9 +47,10 @@ class plugin2DView(wx.Panel):
     """Plugin to display DICOM image, RT Structure, RT Dose in 2D."""
 
     def __init__(self):
-        pre = wx.PrePanel()
+        #pre = wx.PrePanel()
```

Comment:
I think these can be dropped entirely instead of leaving them as comments if it's no longer needed by wxPython 4.